src/plugins/validation.ts

```diff
@@ -132,12 +130,59 @@ const isBetween = (
  * @returns {function(*): (boolean|string)}
  */
 const isDateValid = (
-  message?: string
+    message?: string
 ): ((v: string) => string | true | undefined) => {
   message = message || `Invalid Date`;
   // validate date isn't something like 12/DD/YYYY
   return (v: string) => {
-    return (/^[0-9]*$/.test(v.replaceAll(/\//g, ""))) || message 
+    return (/^[0-9]*$/.test(v.replaceAll(/\//g, ""))) || message
+  };
+};
+
+/**
+ * Validator that validates if input is a valid Email
+ * Returns the error message otherwise.
+ *
+ * @param (string) email as "test@mail.mil"
+ * @returns {function(*): (boolean|string)}
+ */
+const isEmail = (): ((v: string) => string | true | undefined) => {
+  return (v: string) => {
+    if (v !== "") {
+      if (/[a-z0-9]+@[a-z]+\.[a-z]{3}/.test(v)) {
+        return "Please use standard domain format, like ‘@mail.mil’"

```

Comment:
Something off with the logic here
![Screen Shot 2022-04-18 at 2 13 00 PM](https://user-images.githubusercontent.com/90333349/163854119-14913a62-93fc-4d13-b843-4eeed8e60ff2.png)